internal/daemon/cluster/handlers/worker_service_test.go

```diff
@@ -1320,13 +1320,9 @@ func TestRoutingInfo(t *testing.T) {
 			{Key: "tag1", Value: "value1"},
 			{Key: "tag2", Value: "value2"},
 		}
-		assert.Empty(
-			t,
-			cmp.Diff(
-				w1.ConfigTags,
-				server.Tags(expTags),
-			),
-		)
+		for _, v := range expTags {
+			assert.Equal(t, v.Value, w1.ConfigTags[v.Key][0])
+		}
```

Comment:
This is now a weaker assertion. This could pass while allowing for additional tags being returned from LookupWorker. There could be additional keys in `w1.ConfigTags` that are not included in `expTags` as well as additional values for a key. I would suggest that we either change the `expTags` to be in the `worker.Tags` format, or convert from `worker.Tags` to `[]*server.Tag` before making the assertions.